_tuning-your-cluster/availability-and-recovery/workload-management/wlm-feature-overview.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Workload management
+nav_order: 70
+has_children: true
+parent: Availability and recovery
+---
+
+# Workload management
+
+Workload management allows users to group and search network traffic, isolating system resources to prevent the overuse of network resources by specific requests. It offers the following benefits:
```

Comment:
This wording seems wrong, WLM does not group and search the network traffic rather provide support for multitenancy for search traffic

Reply:
The previous description was more accurate and informative
```
Workload management allows users to group search traffic and isolate system resources, preventing resource hogging by specific requests. This ensures fair resource allocation, even for short-lived, low-intensity queries.
This feature offers tenant-level admission control and reactive query management. It can identify and cancel resource-intensive queries when configured thresholds are exceeded, ensuring fair resource allocation.
This feature provides tenant-level isolation within the cluster for search workloads, operating at a node level.
```